demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py

```diff
@@ -53,7 +53,7 @@ def render_frame(self, frame, logits, timers, frame_ind):
         if logits is not None:
             labels, probs = decode_output(logits, self.lables, top_k=self.number_of_predictions,
                                           label_postprocessing=self.postprocessing)
-            print("Frame {}: {} - {:.2f}% -- {:.2f} fps".format(frame_ind, labels[0], probs[0] * 100, inference_time))
+            print("Frame {}: {} - {:.2f}% -- {:.2f}ms".format(frame_ind, labels[0], probs[0] * 100, inference_time))
```

Comment:
"ms" should still be separated from the number by a space.

Reply:
`Incremental timer` has `"{:.2f}ms (±{:.2f}) {:.2f}fps".format(self.avg, self.std, self.fps)`. I removed a space for consistency.

Reply:
Alright.